<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>761</width>
      <height>451</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Lucky Game:</string>
    </property>
    <widget class="QLabel" name="labelBox1">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>81</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>50</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 85, 127);</string>
     </property>
     <property name="text">
      <string>3</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelBox2">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>70</y>
       <width>81</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>50</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 85, 127);</string>
     </property>
     <property name="text">
      <string>4</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelBox3">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>70</y>
       <width>81</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>50</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 255, 127);</string>
     </property>
     <property name="text">
      <string>6</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>220</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>machine money:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>280</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>player money:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditMachineMoney">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>230</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>100</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditPlayerMoney">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>290</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>100</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonRandom">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>350</y>
       <width>191</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Random</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../images/ic_random.webp</normaloff>../images/ic_random.webp</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonNewGame">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>350</y>
       <width>191</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>New Game</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../images/ic_new.webp</normaloff>../images/ic_new.webp</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonExit">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>350</y>
       <width>191</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../images/ic_exit.webp</normaloff>../images/ic_exit.webp</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonExit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>645</x>
     <y>436</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
